docs/src/conf.py

```diff
@@ -25,7 +25,7 @@
 # -- Project information -----------------------------------------------------
 
 project = "Quality-time"
-copyright = "2021-2023, ICTU"  # noqa: A001
+copyright = "2021-2024, ICTU"  # noqa: A001
```

Comment:
Hehe, might as well set it to 2025 now or leave a ticket for next year ;)

Reply:
Done: https://github.com/ICTU/quality-time/issues/10354